backend/internal/events/event.go

```diff
@@ -0,0 +1,80 @@
+package events
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/kyma-incubator/Kyma-Showcase/internal/model"
+	"github.com/kyma-incubator/Kyma-Showcase/internal/utils"
+	"net/http"
+	"os"
+)
+
+//go:generate mockery --name=EventFactory
+// EventFactory defines all event methods
+type EventFactory interface {
+	NewEvent(id string) model.Event
+}
+
+// ImgEvent fulfills the EventFactory interface
+type ImgEvent struct {
+	Img model.Image
+}
+
+// NewEvent defines the values for Event
+func (e *ImgEvent) NewEvent(id string) model.Event {
+	return model.Event{
+		Source:           "kyma-showcase",
+		SpecVersion:      "1.0",
+		EventTypeVersion: "v1",
+		Data:             e.Img.ID,
+		Id:               id,
+		DataContentType:  "application/json",
+		EventType:        os.Getenv("EVENT_TYPE"),
+	}
+}
+
+// EventHandler for event
+type EventHandler struct {
+	eventFactory EventFactory
+	idGenerator  utils.IdGenerator
+}
+
+// NewEventHandler returns handler for event
+func NewEventHandler(event EventFactory, idGenerator utils.IdGenerator) EventHandler {
+	return EventHandler{
+		eventFactory: event,
+		idGenerator:  idGenerator,
+	}
+}
+
+//go:generate mockery --name=EventHelper
+// EventHelper helps to mock SendEvent method in tests
+type EventHelper interface {
+	SendEvent() error
+}
+
+// SendEvent creates event and sends a request that triggers lambda
+func (e EventHandler) SendEvent() error {
```

Comment:
`SendEvent` is not descriptive enough. I think the name should tell what kind of event we send.